scripts/Start-CleanMachineTesting.ps1

```diff
@@ -0,0 +1,33 @@
+<#
+.SYNOPSIS
+	Installs Windows Sandbox feature if needed and then sets up a clean machine test enviroment
+.DESCRIPTION
+	Starts Windows Sanbox instance and runs a powershell script to setup dependencies and a repo to test gitextensions on a clean machine.
+.NOTES
+	See https://learn.microsoft.com/en-us/windows/security/application-security/application-isolation/windows-sandbox/windows-sandbox-overview
+	to learn about windows sandbox. 
+	Caveat: You must have ran a build for this to fully work.  If artifacts folder is clean, there will be errors.
+#>
+
+#required to turn on windows feature
+#requires -RunAsAdministrator
+function StartWSB() {
+	#Include PSR( problem steps recorder)
+	Copy-Item  "$env:SystemRoot\system32\psr.exe" "$PSScriptRoot\wsb\"
+	Start-Process .\Sandbox.wsb
```

Comment:
This requires to start the script in the repo root.
Rather add a "Push-Location" relative to this script?

Reply:
https://dev.to/taijidude/psscriptroot-is-your-friend-136p

It gets the folder where the running script exists. So this is making sure it has correct path.